web/libs/editor/src/stores/Annotation/Annotation.js

```diff
@@ -339,6 +340,23 @@ export const Annotation = types
         }
       : {},
   )
+  .views((self) => ({
+    // experiment to display review buttons in Quick View
+    get canBeReviewed() {
+      const store = self.store;
+
+      return isFF(FF_REVIEWER_FLOW)
+        // not a current user — we can only review others' annotations
+        && self.user?.email
+        && store.user?.email !== self.user?.email
+        // we have this only in LSE
+        && store.hasInterface("annotations:history")
```

Comment:
most probably it's also added as part of default interfaces for LSO, so we should find another way to distinguish LSO and LSE. @Gondragos do you have any ideas?

Reply:
What about just define new interface for exactly this purpouse?

Reply:
Or.... there is an option
```js
getEnv(self).events.hasEvent("acceptAnnotation")
```